src/mixins/legal-api-mixin.ts

```diff
@@ -6,34 +6,35 @@ import { NOT_FOUND } from 'http-status-codes'
 
 // Interfaces
 import { ActionBindingIF, IncorporationFilingIF } from '@/interfaces'
+import { FilingTypes } from '@/enums'
 
 /**
  * Mixin that provides integration with the Legal API.
  */
 @Component({})
 export default class LegalApiMixin extends Vue {
-  readonly INCORPORATION_APPLICATION = 'incorporationApplication'
-
+  @Getter getBusinessId!: string
```

Comment:
I believe business number is a number in the db.  is there an issue if it isn't a string?

Reply:
This property is referring to the business identifier, which is a string in the response or request schemas. 
ie `BC1234567`

Reply:
![Screen Shot 2021-10-04 at 9 24 03 AM](https://user-images.githubusercontent.com/53542131/135888385-5786f4ac-fe88-4478-9b3d-81cba4fd451b.png)

Reply:
Sorry, crummy screenshot there. Doesn't give much context 😆 
But yes, straight from schema for the business.

Reply:
oh ok, yeah that needs a string.